static/testdata/aggregates/single-aggregate-app/aggregate.go

```diff
@@ -0,0 +1,49 @@
+package app
+
+import (
+	"github.com/dogmatiq/dogma"
+	"github.com/dogmatiq/dogma/fixtures"
+)
+
+// Aggregate is an aggregate used for testing.
+type Aggregate struct{}
+
+// ApplyEvent updates the aggregate instance to reflect the occurrence of an
+// event that was recorded against this instance.
+func (Aggregate) ApplyEvent(m dogma.Message) {}
+
+// AggregateHandler is a test implementation of dogma.AggregateMessageHandler.
+type AggregateHandler struct{}
+
+// New returns a new account instance.
+func (AggregateHandler) New() dogma.AggregateRoot {
+	return Aggregate{}
+}
+
+// Configure configures the behavior of the engine as it relates to this
+// handler.
+func (AggregateHandler) Configure(c dogma.AggregateConfigurer) {
+	c.Identity("<aggregate>", "ef16c9d1-d7b6-4c99-a0e7-a59218e544fc")
+
+	c.ConsumesCommandType(fixtures.MessageA{})
+	c.ConsumesCommandType(fixtures.MessageB{})
+	c.ConsumesCommandType(fixtures.MessageC{})
+
+	c.ProducesEventType(fixtures.MessageD{})
+	c.ProducesEventType(fixtures.MessageE{})
+	c.ProducesEventType(fixtures.MessageF{})
```

Comment:
In the interest of reducing the size and complexity of the tests, you could reduce the number of message types used here to just 2 of each instead of 3.

Likewise, since this test covers the case where there are multiple message types the other tests don't need to duplicate that; they could just have a single command and a single event.

I am only suggesting this to reduce clutter, no big deal if you want to retain them.

Reply:
Done, the number of messages reduced in https://github.com/dogmatiq/configkit/pull/123/commits/f5ca4975c9c602e436921b0e95a36264bab66712.